src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1250,6 +1252,16 @@ private bool IsQueryMatch(string query, string symbolName)
             });
         }
 
+        protected async Task HandleFoldingRangeRequest(
+            FoldingRangeParams foldingParams,
+            RequestContext<FoldingRange[]> requestContext)
+        {
+            var result = Fold(
+                foldingParams.TextDocument.Uri);
```

Comment:
When lines are this short, I see no reason to wrap to a second line.

Reply:
I think I had previously passed in more params.  Will fix.

Reply:
Fixed